include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -674,6 +674,17 @@ def LLVM_FMulAddOp : LLVM_Op<"intr.fmuladd", [NoSideEffect]>,
   }];
 }
 
+def LLVM_prefetch : LLVM_ZeroResultOp<"intr.prefetch">,
```

Comment:
The classes around follow the naming pattern LLVM_CamelCasedOp, please make sure we keep it that way.